hikari/utilities/net.py

```diff
@@ -53,7 +53,13 @@ async def generate_error_response(response: aiohttp.ClientResponse) -> errors.HT
     if cls is not None:
         try:
             json_body = await response.json()
-            return cls(real_url, response.headers, raw_body, json_body["message"], json_body["code"])  # type: ignore[call-arg]
+            return cls(
```

Comment:
see thread at https://discordapp.com/channels/574921006817476608/643543719391264780/758435631834595339 for discussion we had after you left tonight.

Reply:
Fixed :+1: